AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.java

```diff
@@ -1592,13 +1592,20 @@ private boolean onImagePaste(EditText editText, Uri uri) {
     @Nullable
     private String loadImageIntoCollection(Uri uri) throws IOException {
         //noinspection PointlessArithmeticExpression
+        String fileName;
         final int oneMegabyte = 1 * 1000 * 1000;
         String filename = ContentResolverUtil.getFileName(getContentResolver(), uri);
         InputStream fd = getContentResolver().openInputStream(uri);
 
         Map.Entry<String, String> fileNameAndExtension = FileUtil.getFileNameAndExtension(filename);
 
-        File clipCopy = File.createTempFile(fileNameAndExtension.getKey(), fileNameAndExtension.getValue());
+        if (checkFilename(fileNameAndExtension)) {
+            fileName = String.format("%s-name", fileNameAndExtension.getKey());
+        }else {
```

Comment:
spacing nit, trivial

```suggestion
        } else {
```